components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -132,8 +132,9 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
             return <ListItemIcon style={{ color: getIconColor() }}>{icon}</ListItemIcon>;
         } else if (!hidePadding) {
             return (
+                // a dummy component to maintain the padding
                 <ListItemAvatar>
-                    <Avatar style={{ backgroundColor: 'transparent' }} />
+                    <Avatar style={{ opacity: 0 }} />
```

Comment:
I donn't want it to display an `<Avatar />` by default when there is no icon and hidePadding is set to false. But taking `<Avatar />` out will cause `< ListItemAvatar />` to complain because a child is required (another required children!)